app/elements/cardlist-api/card-assignment-create-panel.html

```diff
@@ -75,6 +75,9 @@
       <span class="error" hidden$="{{!error}}">One of the fields is incorrect!</span>
       <paper-button raised on-tap="submit">Submit</paper-button>
     </div>
+
+    <h2>Upload Multiple</h2>
+    <file-upload on-success="fileFinished" target="/v1/courses/lectures" droppable raised>Select file</file-upload>
```

Comment:
Change method to POST. Add headers for token generate and probably put this in `file-upload-wrapper`